st/clitests/awss3api_spec.py

```diff
@@ -99,6 +134,69 @@ def delete_object_list_file(file_name):
 AwsTest('Aws can delete object').delete_object("seagatebucket", "3kfile")\
     .execute_test().command_is_successful()
 
+#*********** Verify aws s3api list-objects-v2 ***********
+# Setup objects in bucket
+obj_list = ['asdf', 'quax/t3/key.log', 'foo', 'quax/t1/key.log', 'boo', 'zoo/p2/key.log', 'zoo/p7/key.log']
+# Create above objects in existing bucket 'seagatebucket'
+for x in obj_list:
+    AwsTest(('Aws upload object:%s' % x)).put_object("seagatebucket", "1kfile", 1000, key_name=x)\
+        .execute_test().command_is_successful()
+
+# Start Listing objects using V2 scheme
+# Test #1: List objects using --start-after, --max-items and --starting-token
+list_v2_options = {"start-after":"boo", "max-items":2}
+result = AwsTest('Aws list-objects-v2: start-after and max-items').list_objects_v2("seagatebucket", **list_v2_options)\
+    .execute_test().command_is_successful().command_response_should_have("quax/t1/key.log")\
+    .command_response_should_have("foo")
+
+#TODO print("Result=", str(result.status.stdout))
+# Process result set
+lv2_response = get_aws_cli_object(result.status.stdout)
+# Get next-token from lv2_response
+if ("next_token" in lv2_response.keys()):
+    Next_token = lv2_response["next_token"]
+
+# Get keys from lv2_response
+obj_keys =  lv2_response["keys"]
+
+# Assert for key existence
+assert "foo" in obj_keys, "Key \"foo\" is expected to be in the result"
```

Comment:
should we add negative test also :- 

assert key not in obj_keys ?

Reply:
Will try to add -ve test case

Reply:
_ve test case added below.